### PR TITLE
Energy Star AI - Initial Proposal

### PR DESCRIPTION
**Pull request by sashavor**
A brief write up of the general idea, motivation and initial results for the "Energy Star AI" project.

Open to suggestions of other stuff to add (maybe images or figures?)

### CODE REVIEW

File: energystarai-intro.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+## Energy Star AI Project
+
+Every time we query an AI model, we also use energy, whether it be locally on our computer or on a server in the cloud. The amount of energy that we use will depend on the characteristics of the model, such as its size and architecture, and the way in which it's deployed, i.e. the optimizations and engineering choices made.
+
+The aim of the [Energy Star AI project](https://huggingface.co/EnergyStarAI) is to develop an Energy Star rating system for AI model deployment that will guide members of the community in choosing models for different tasks based on their energy efficiency and to analyze the effect of implementation choices on the downstream energy usage of different models.
+
+## Tasks and Models
+
+Since the original Energy Star ratings were developed to span a variety of use cases and consumer products, we picked 10 popular [tasks](https://huggingface.co/tasks) spanning both language and computer vision, and including multi-modal tasks. By testing a variety of models across different tasks, we aim to cover different use cases and AI applications that are relevant to different groups:
```
Comment (regisss) [inline energystarai-intro.md:25]: > spanning both language and computer vision, and including multi-modal tasks

Is it worth mentioning audio too?
edit: dunno, I admit it's pretty obvious in the list right below

**Review by regisss — COMMENTED** [review]
LGTM!

File: _blog.yml [hunk 2]
```diff
@@ -3935,3 +3935,15 @@
     - research
     - leaderboard
     - LLM
+    
+- local: energystarai_intro
```
Comment (julien-c) [inline _blog.yml:3939]:
```suggestion
- local: energy-star-ai
```
more consistent name/URL

File: energystarai-intro.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## Energy Star AI Project
+
+Every time we query an AI model, we also use energy, whether it be locally on our computer or on a server in the cloud. The amount of energy that we use will depend on the characteristics of the model, such as its size and architecture, and the way in which it's deployed, i.e. the optimizations and engineering choices made.
+
+The aim of the [Energy Star AI project](https://huggingface.co/EnergyStarAI) is to develop an Energy Star rating system for AI model deployment that will guide members of the community in choosing models for different tasks based on their energy efficiency and to analyze the effect of implementation choices on the downstream energy usage of different models.
```
Comment (julien-c) [inline energystarai-intro.md:21]:
```suggestion
The aim of the [Energy Star AI project](https://huggingface.co/EnergyStarAI) is to develop an Energy Star rating system for AI model deployment that will guide members of the community in choosing models (and ways to run them) for different tasks based on their energy efficiency and to analyze the effect of implementation choices on the downstream energy usage of different models.
```

File: _blog.yml [hunk 4]
```diff
@@ -3935,3 +3935,15 @@
...
+  title: "Energy Star Ratings for AI Models - a Proposal"
+  thumbnail: /blog/assets/energystarai-intro/thumbnail.png
+  author: sasha
+  guest: true
```
[the remark this answers is not in the document]
Comment (sashavor) [reply _blog.yml:3953]: I think that @yjernite intended for this to be a community post (and not a "HF-branded" one), if I'm not mistaken

File: energystarai-intro.md (hunk 1 above)
Comment (sashavor) [reply energystarai-intro.md:25]: sure, I can add! :hugs:

File: energystarai-intro.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+# Energy Star Ratings for AI Models
+
+## Background and Inspiration - EPA Energy Star Ratings
+
+Every time we use an electronic device like a computer, a phone, or even a washing machine, we are using energy to power the device; and depending on how that energy is generated, this can result in greenhouse gas (GHG) emissions. In 1992, the U.S. Environmental Protection Agency (EPA) launched the Energy Star program as a way to set energy efficiency specifications for different types of devices and help consumers make informed decisions. In the last 30 years, the Energy Star program has saved billions of metric tons of greenhouse gas emissions and has come to encompass dozens of product categories, from data centers to dishwashers.
+
+## Energy Star AI Project
+
+Every time we query an AI model, we also use energy, whether it be locally on our computer or on a server in the cloud. The amount of energy that we use will depend on the characteristics of the model, such as its size and architecture, and the way in which it's deployed, i.e. the optimizations and engineering choices made.
```
Comment (yjernite) [inline energystarai-intro.md:19]:
```suggestion
For each query sent to an AI model, we also use energy, whether it be locally on our computer or on a server in the cloud. The amount of energy that we use will depend on the characteristics of the model, such as its size and architecture, and the way in which it's deployed, i.e. the optimizations and engineering choices made.
```

File: energystarai-intro.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+The aim of the [Energy Star AI project](https://huggingface.co/EnergyStarAI) is to develop an Energy Star rating system for AI model deployment that will guide members of the community in choosing models (and ways to run them) for different tasks based on their energy efficiency and to analyze the effect of implementation choices on the downstream energy usage of different models.
+
```
Comment (yjernite) [inline energystarai-intro.md:22]:
```suggestion

This article presents our initial work toward developing a broadly applicable energy rating system for AI models to be used **in conjunction with** other measures, including for example measures of task-specific performance. The choices made here represent our approach to setting a solid foundation for the project and are meant to evolve to increase its breadth and robustness in subsequent versions; we welcome early feedback to that end!
```

**Review by yjernite — COMMENTED** [review]
Added some comments here for convenience - as discussed on Slack let's move this to an Article on the Hub

File: _blog.yml (hunk 4 above)
[the remark this answers is not in the document]
Comment (julien-c) [reply _blog.yml:3953]: but then you need some community co-authors:)

**Pull request closed without merging**